level_1/valid_privacy_information.py

```diff
@@ -0,0 +1,52 @@
+# 개인정보 수집 유효 기간 문제
+# 구현 문제
+# 시간 복잡도 : O(n)
+import re
+
+
+def solution(today: str, terms: list, privacies: list) -> list:
+    answer = []
+    terms_valid = make_terms_dictionary(terms)
+    for privacy_index, privacy in enumerate(privacies):
+        valid_privacy = make_valid_privacy(privacy, terms_valid)
+        if is_destroy(today, valid_privacy):
+            answer.append(privacy_index + 1)
+    return answer
+
+
+# 약관과 유효기간을 dictionary 형태로 만든다.
+def make_terms_dictionary(terms: list) -> dict:
+    term_valid = {}
+    for term in terms:
+        tos = term.split()
+        term_valid[tos[0]] = int(tos[1])
+    return term_valid
+
+
+# 개인 정보 수집 일자로 만료 기간을 만들어주는 함수
+def make_valid_privacy(privacy: str, term_valid: dict):
+    year, month, day, term = map(str, re.split('[ .]', privacy))
+    year, month, day, term = int(year), int(month), int(day), term_valid[term]
```

Comment:
```suggestion
    date, term = map(str, re.split('[ .]', privacy
    year, month, day = map(int, date)
```
이런 식으로 map을 보다 활용할 수 있지 않을까??